tests/test_sim_results/electric_blue/test_volume_timeseries.py

```diff
@@ -0,0 +1,203 @@
+from typing import List
+
+import pytest
+from pendulum import DateTime
+
+from gsy_framework.enums import AggregationResolution
+from gsy_framework.sim_results.electric_blue.aggregate_results import (
+    ForwardDeviceStats)
+from gsy_framework.sim_results.electric_blue.volume_timeseries import (
+    FORWARD_MARKET_TYPES, AssetVolumeTimeSeries)
+from tests.forward_markets.test_aggregated_profile import \
+    get_sum_of_ssp_profile_range_values as sum_of_energy
+
+
+@pytest.fixture
+def forward_time_series():
+    """Return a generator that produces ForwardDeviceStats objects for testing purposes."""
+    def func():
+        for market_type in FORWARD_MARKET_TYPES:
+            yield market_type, ForwardDeviceStats(
+                "UUID", DateTime(2020, 1, 1),
+                DateTime(2020, 1, 1),
+                total_energy_sold=1,
+                total_energy_bought=1
+            )
+    return func()
+
+
+class TestAssetVolumeTimeSeries:
+    @staticmethod
+    def check_time_slot_pairs(time_slots: List, resolution: AggregationResolution):
+        """Check the returned value of _adapt_time_slot is equal to the expected value."""
+        volume_time_series = AssetVolumeTimeSeries(
+            "UUID", 1, resolution)
```

Comment:
Here and in the rest of the module, I'd recommend using named parameters to instantiate this class:
```suggestion
        volume_time_series = AssetVolumeTimeSeries(
            asset_uuid="UUID", 
            asset_capacity=1, 
            resolution=resolution)
```
This makes it easier to refactor in the future. 
Using positional parameters makes sense in some cases (e.g. when the name of the function makes it obvious what they represent, like `multiply(2, 4)`) but in this case we have no idea what `1` is, for example.

Reply:
Fixed.